test/parallel/test-repl-definecommand.js

```diff
@@ -34,8 +34,9 @@ r.defineCommand('say2', function() {
   this.displayPrompt();
 });
 
+
 inputStream.write('.help\n');
-assert(/\n\.say1     help for say1\n/.test(output),
```

Comment:
because  add `.load-editor` command need more space for fomat
```
> .help
.break         Sometimes you get stuck, this gets you out
.clear         Break, and also clear the local context
.editor        Enter editor mode
.exit          Exit the repl
.help          Print this help message
.load          Load JS from a file into the REPL session
.load-editor   Load a file and enter editor mode
.save          Save all evaluated commands in this REPL session to a file
.say1          help for say1
.say2
>
```